src/greeklt/greek_elot_transliteration.py

```diff
@@ -0,0 +1,231 @@
+    string = remove_accentuation(string, 1)
+    lowercase = {
+        'α': 'a',
+        'β': 'v',
+        'γ': 'g',
+        'δ': 'd',
+        'ε': 'e',
+        'ζ': 'z',
+        'η': 'i',
+        'θ': 'th',
+        'ι': 'i',
+        'κ': 'k',
+        'λ': 'l',
+        'μ': 'm',
+        'ν': 'n',
+        'ξ': 'x',
+        'ο': 'o',
+        'π': 'p',
+        'ρ': 'r',
+        'σ': 's',
+        'τ': 't',
+        'υ': 'y',
+        'φ': 'f',
+        'χ': 'ch',
+        'ψ': 'ps',
+        'ω': 'o'
+    }
+    caps = {
+        'Α': 'A',
+        'Β': 'V',
+        'Γ': 'G',
+        'Δ': 'D',
+        'Ε': 'E',
+        'Ζ': 'Z',
+        'Η': 'I',
+        'Θ': 'TH',
+        'Ι': 'I',
+        'Κ': 'K',
+        'Λ': 'L',
+        'Μ': 'M',
+        'Ν': 'N',
+        'Ξ': 'X',
+        'Ο': 'O',
+        'Π': 'P',
+        'Ρ': 'R',
+        'Σ': 'S',
+        'Τ': 'T',
+        'Υ': 'Y',
+        'Φ': 'F',
+        'Χ': 'CH',
+        'Ψ': 'PS',
+        'Ω': 'O'
+    }
+    # Simple digraphs with no extra rules or edge cases
+    # No need for mixed casing "Γγ" or "Γξ" against Greek phonology
+    el_simple_digraphs = [
+        'γγ',
+        'γξ',
+        'γχ',
+        'ου'
+    ]
+    eng_simple_digraphs = [
+        'ng',
+        'nx',
+        'nch',
+        'ou'
+    ]
+    el_simple_cap_digraphs = [
+        "ΓΓ",
+        "ΓΞ",
+        "ΓΧ",
+        "ΟΥ"
+    ]
+    eng_simple_cap_digraphs = [
+        "NG",
+        "NX",
+        "NCH",
+        "OU"
+    ]
+
+    el_mono_digraph_sub = [
+        "TH",
+        "CH",
+        "PS"
+    ]
+#   Accent based digraphs
+#    el_low_acc_digraphs = [
+#        "άυ",
+#        "αϋ",
+#
+#        "έυ",
+#        "εϋ",
+#
+#        "ήυ",
+#        "ηϋ"
+#
+#    ]
+#    el_mix_acc_digraphs = [
+#        "Άυ",
+#        "Αϋ",
+#
+#        "Έυ",
+#        "Εϋ",
+#
+#        "Ήυ",
+#        "Ηϋ"
+#    ]
+#    el_cap_acc_digraphs = [
+#        "ΆΥ",
+#        "ΑΫ",
+#
+#        "ΈΥ",
+#        "ΕΫ",
+#
+#        "ΉΥ",
+#        "ΗΫ"
+#    ]
+    el_mp_digraph = [
+        "ΜΠ",
+        "Μπ",
+        "μπ"
+    ]
+    eng_mp_digraph_0 = [
+        "B",
+        "B",
+        "b"
+    ]
+    eng_mp_digraph_1 = [
+        "MP",
+        "Mp",
+        "mp"
+    ]
+    el_xu_digraphs = [
+        "αυ",
+        "ευ",
+        "ηυ"
+    ]
+    eng_xu_digraphs_v = [
+        "av",
+        "ev",
+        "iv"
+    ]
+    eng_xu_digraphs_f = [
+        "af",
+        "ef",
+        "if"
+    ]
+    xu_sound_modifiers_v = [
+
+        "β",
+        "γ",
+        "δ",
+        "ζ",
+        "λ",
+        "μ",
+        "ν",
+        "ρ",
+        
+        "α",
+        "ε",
+        "η",
+        "ι",
+        "ο",
+        "υ",
+        "ω"
+    ]
+#   +empty space (accounted for in code)
+    xu_sound_modifiers_f = [
+
+        "θ",
+        "κ",
+        "ξ",
+        "π",
+        "σ",
+        "τ",
+        "φ",
+        "χ",
+        "ψ"
+    ]
```

Comment:
I would preffer all list pairs tyo be converted into dicts but it's more of a readability enchancement!